web/scene.js

```diff
@@ -101,26 +104,22 @@ Scene.prototype.printLoop = function printLoop() {
         } else if (indent < this.indent) {
             this.dedent(indent);
         }
-        if (this.temps.fakeChoiceLines && this.temps.fakeChoiceLines[this.lineNum]) {
-          this.rollbackLineCoverage();
-          this.lineNum = this.temps.fakeChoiceEnd;
-          this.rollbackLineCoverage();
-          delete this.temps.fakeChoiceEnd;
-          delete this.temps.fakeChoiceLines;
-          continue;
+        // Ability to end a choice #option without goto is guarded by implicit_control_flow variable
+        if (this.temps._choiceEnds[this.lineNum] && 
+                (this.stats["implicit_control_flow"] || this.temps._fakeChoiceDepth > 0)) {
+            // Skip to the end of the choice if we hit the end of an #option
+            this.rollbackLineCoverage();
+            this.lineNum = this.temps._choiceEnds[this.lineNum];
+            this.paragraph();
```

Comment:
This call to paragraph breaks a unit test. ("ant unittest")

In particular, in this sample:
```
*fake_choice
  #foo
    Foo!
  #bar
    Bar!
baz
```

ChoiceScript should render "Foo! baz" but with the addition of the paragraph call, baz is rendered in a separate paragraph.

I can just nuke the line on my end, but perhaps you added it for some reason I should be aware of?